x-pack/plugins/asset_manager/server/plugin.ts

```diff
@@ -61,11 +68,15 @@ export class AssetManagerServerPlugin implements Plugin<AssetManagerServerPlugin
     }
 
     // create/update assets-* index template
-    upsertTemplate({
-      esClient: core.elasticsearch.client.asInternalUser,
-      template: assetsIndexTemplateConfig,
-      logger: this.logger,
-    });
+    if (this.config.lockedSource === 'assets') {
```

Comment:
Don't install assets template since we don't dynamically switch between sources yet

Reply:
Most of the integration tests verify the features related to asset documents and thus expect the `assets-*` template, since we don't install it anymore they are now failing with that change. I'm considering creating two ftr configs, one with a kibana that reads from signals and the other from assets, and enabling/disabling test suites depending on the running configuration. FOr now only the _signals_ configuration would be running. @neptunian @jasonrhodes any thoughts on this ?

Reply:
That sounds like a good plan!